source/dem/from_file_insertion.cc

```diff
@@ -0,0 +1,188 @@
+#include <core/utilities.h>
+
+#include <dem/from_file_insertion.h>
+
+
+using namespace DEM;
+
+template <int dim>
+FromFileInsertion<dim>::FromFileInsertion(
+  const DEMSolverParameters<dim> &dem_parameters,
+  const std::vector<std::shared_ptr<Distribution>>
+    &distribution_object_container)
+  : Insertion<dim>(distribution_object_container)
+  , remaining_particles_of_each_type(
+      dem_parameters.lagrangian_physical_properties.number.at(0))
+{
+  // Initializing current inserting particle type
+  this->current_inserting_particle_type = 0;
+
+  file_name = dem_parameters.insertion_info.insertion_particles_file;
+}
+
+// The main insertion function. Insert_global_function is used to insert the
```

Comment:
```suggestion
// The main insertion function. particle_handler.insert_global_particles is used to insert the
```

Reply:
I don't think adding "particle_handler.insert_global_particles is used to insert the particles" is relevant in the function description in my opinion

Reply:
agreed